rust-runtime/aws-smithy-checksums/Cargo.toml

```diff
@@ -1,6 +1,6 @@
 [package]
 name = "aws-smithy-checksums"
-version = "0.60.13"
+version = "0.70.0"
```

Comment:
Not that it really matters, but I would expect the next version to be `v0.61.0`

Reply:
I hit this with the flexibile checksums work, `0.61.0` was already published and yanked: https://crates.io/crates/aws-smithy-checksums/versions

But I would probably expect `0.62.0`

Reply:
Good point.